### PR TITLE
hydro2023 fix

### PR DESCRIPTION
**Pull request by bczernecki**
Hot fixes for monthly and daily hydro dataset that include different encodings and seperator characters.

### CODE REVIEW

File: R/clean_metadata_hydro.R [hunk 1]
```diff
@@ -24,22 +24,5 @@ clean_metadata_hydro = function(address, interval) {
   if (interval == "daily") {
     b = data.frame(parameters = a[1:10])
   }
-  if (interval == "semiannual_and_annual") {
```
[the remark this answers is not in the document]
Comment (bczernecki) [reply R/clean_metadata_hydro.R:27]: Raczej nie. Od biedy można przywrócić ze starszych commitów.

**Comment by bczernecki** [comment]
@Nowosad ready for re-review

**Comment by Nowosad** [comment]
``` r
devtools::load_all()
#> ℹ Loading climate
h2022 = hydro_imgw_monthly(2022)
#> /tmp/Rtmphey1As/file196d36654abf4
#> /tmp/Rtmphey1As/file196d32351d8f5
#> /tmp/Rtmphey1As/file196d356036c71
h2023 = hydro_imgw_monthly(2023)
#> /tmp/Rtmphey1As/file196d3341d7c9e
#> /tmp/Rtmphey1As/file196d31bb006c6
#> /tmp/Rtmphey1As/file196d36f58fb13
h2024 = hydro_imgw_monthly(2024)
#> /tmp/Rtmphey1As/file196d34142d196
#> Problems with downloading data. Run function with argument allow_failure = FALSE to see more details
```

One question @bczernecki -- no data for 2024 exist yet?

**Comment by bczernecki** [comment]
> One question @bczernecki -- no data for 2024 exist yet?

Not yet

**Comment by bczernecki** [comment]
> One more question -- why `meteo_imgw_datastore()` returns a data.table and the rest (?) of the functions return a regular data frame? Is there a reason for this inconsistency?

This is the largest of all datasets (10-min time resolution) and it is based on data.table which is significantly faster. On the other hand it has consistent structure contrary to other IMGW datasets which varies in terms of columns order, etc... so this is exception only for this dataset. However, we can think about wrapping it into `as.data.frame()` before `return` statement

**Review by Nowosad — APPROVED** [review]

**Pull request merged**